environments/aks/preview.tfvars

```diff
@@ -1,5 +1,5 @@
 cluster_count                      = 2
-kubernetes_cluster_version         = "1.21.7"
+kubernetes_cluster_version         = "1.23"
```

Comment:
Just update this - no need for azure-pipeline update

Reply:
Done.